src/components/add-cash/AddCashForm.js

```diff
@@ -35,14 +36,24 @@ const AddCashForm = ({
     params?.amount ? params?.amount?.toString() : ''
   );
 
-  const handlePurchase = useCallback(() => {
-    analytics.track('Submitted Purchase', {
-      category: 'add cash',
-      label: currency,
-      value: Number(value),
-    });
-    return onPurchase({ currency, value });
-  }, [currency, onPurchase, value]);
+  const handlePurchase = useCallback(async () => {
+    if (paymentSheetVisible) return;
+    try {
+      analytics.track('Submitted Purchase', {
+        category: 'add cash',
+        label: currency,
+        value: Number(value),
+      });
+      console.log('visible!');
```

Comment:
Can you remove it?

Reply:
I did remove it and forgot to commit! 🤦 
![Screen Shot 2020-10-16 at 2 57 42 AM](https://user-images.githubusercontent.com/1247834/96223111-653eff00-0f5b-11eb-9311-2a911c13f842.png)